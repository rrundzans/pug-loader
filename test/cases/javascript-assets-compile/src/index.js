const tmpl = require('Template/widget.pug');

console.log(tmpl({
  data: {
    items: {
      a: 123,
      b: 'abc'
    }
  }
}));