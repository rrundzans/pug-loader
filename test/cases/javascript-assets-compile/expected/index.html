<h1>Hello World!</h1><img class="a1" src="assets/images/image.f78b30f4.jpeg"><img class="a2" src="assets/images/image.f78b30f4.jpeg"><img class="a3" src="assets/images/image.f78b30f4.jpeg"><img class="a3" src="assets/images/image.f78b30f4.jpeg"><img class="a4" src="assets/images/image.f78b30f4.jpeg"><img class="b1" src="assets/images/image.f78b30f4.jpeg"><img class="b2" src="assets/images/image.f78b30f4.jpeg"><img class="c1" src="assets/images/image.f78b30f4.jpeg"><img class="c2" src="assets/images/image.f78b30f4.jpeg"><img class="d1" src="assets/images/image.f78b30f4.jpeg"><img class="d2" src="assets/images/image.f78b30f4.jpeg"><img class="d3" src="assets/images/image.f78b30f4.jpeg"><img class="d4" src="assets/images/image.f78b30f4.jpeg"><img class="e1" src="assets/images/image.f78b30f4.jpeg"><img class="e1" src="assets/images/image.f78b30f4.jpeg"><div class="a">123</div><div class="b">abc</div><p>Hello green (#0f0)!</p><div class="first"><i>red</i><i>green</i><i>blue</i></div><div class="second"><h3>Red Green Blue</h3><i>red</i><i>green</i><i>blue</i><h3>Rose Violet Azure</h3><i>rose</i><i>violet</i><i>azure</i></div>